crates/astria-sequencer/src/app.rs

```diff
@@ -639,6 +639,10 @@ impl App {
             .get_block_deposits()
             .await
             .context("failed to get block deposits in end_block")?;
+        debug!(
+            deposits = %telemetry::display::json(&DisplayDeposits(&deposits)),
```

Comment:
mentioned below, just do:
```suggestion
            deposits = %telemetry::display::json(&deposits),
```